src/ui/components/atoms/Toggle.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import styles from './Toggle.module.scss';
+
+export interface ToggleProps {
+  label?: string;
+  disabled?: boolean;
+  onChange?: (checked: boolean) => void;
+}
+
+function Toggle({ label = '', disabled = false, onChange }: ToggleProps) {
+  const [checked, setChecked] = useState(false);
+
+  const handleToggle = () => {
+    if (!disabled) {
+      setChecked(!checked);
+      if (onChange) onChange(!checked);
+    }
+  };
+
+  return (
+    <div className={`${styles.container} ${disabled ? styles.disabled : ''}`}>
```

Comment:
please use this approach https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx